examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,11 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>Add theano and xarray here, I added how to do that to the style guide: https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>